.github/workflows/after-merge.yml

```diff
@@ -0,0 +1,157 @@
+# Run after merge to trunk
+on:
+  push:
+    branches: [master] # $default-branch
+
+jobs:
+  build:
```

Comment:
Is there a way to keep this job in sync with the `test-all-packages.yml` version?

Reply:
I introduced a new composite action.  That should help, though it is more indirect.